### PR TITLE
PP-3686 Make pay-selfservice test directory great again

### PR DESCRIPTION
**Pull request by brid**
## WHAT
Various commits to clean up the state of play inside pay-selfservice prior to more extensive work to integrate full contract testing.

### CODE REVIEW

File: test/unit/utils/paginator_tests.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var Paginator = require('../../app/utils/paginator.js')
+var Paginator = require('../../../app/utils/paginator.js')
```
Comment (jonheslop) [inline test/unit/utils/paginator_tests.js]: Lol

File: test/unit/utils/util_dates_tests.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var path = require('path')
-require(path.join(__dirname, '/../test_helpers/html_assertions.js'))
+require(path.join(__dirname, '/../../test_helpers/html_assertions.js'))
 var assert = require('assert')
-var dates = require(path.join(__dirname, '/../../app/utils/dates.js'))
+var dates = require(path.join(__dirname, '/../../../app/utils/dates.js'))
```
Comment (maxcbc) [inline test/unit/utils/util_dates_tests.js]: we should get rid of the path.joins in require statements.

File: test/unit/utils/utils_filters_test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 var path = require('path')
-var filters = require(path.join(__dirname, '/../../app/utils/filters.js'))
+var filters = require(path.join(__dirname, '/../../../app/utils/filters.js'))
```
Comment (maxcbc) [inline test/unit/utils/utils_filters_test.js]: we should get rid of the path.joins in require statements.

File: test/unit/utils/util_correlation_header_test.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 var path = require('path')
 var assert = require('assert')
-var withCorrelationHeader = require(path.join(__dirname, '/../../app/utils/correlation_header.js')).withCorrelationHeader
+var withCorrelationHeader = require(path.join(__dirname, '/../../../app/utils/correlation_header.js')).withCorrelationHeader
```
Comment (maxcbc) [inline test/unit/utils/util_correlation_header_test.js]: we should get rid of the path.joins in require statements.

**Pull request merged**